_pages/presentation_materials.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+title: "Apache Beam Presentation Materials"
+permalink: /presentation-materials/
+---
+
+Are you interested in learning more about Apache Beam or giving a talk about Apache Beam? Excellent! The Apache Beam community has created this collection of materials to help you get started. The community periodically reviews these materials to ensure they are accurate and up-to-date.
+
+<div class="panel panel-default">
+  <div class="panel-body">
+    <iframe src="https://drive.google.com/embeddedfolderview?id=0B-IhJZh9Ab52a3JLVXFWMDltcHM#list" width="700" height="300" frameborder="0"></iframe>
+  </div>
+</div>
+
+### Using these materials
+These materials are free to use! If you do use these materials, please remember to give credit to the authors of the original materials. These materials include speaker notes and you can view them by clicking **`View`** and selecting **`Show speaker notes`**. A set of presentations are have `[Long]` or `[Short]` in their name. Some topics, such as the *Apache Beam Model*, have two different presentations (a long and short version) depending on your needs and time constraints.
+
+If you use these materials and questions arise:
+
+  * If anyone asks questions to which you are unsure of the answer, please do not hesitate to contact or direct the question to the [`user@beam.incubator.apache.org`](mailto:user@beam.incubator.apache.org) mailing list. The Apache Beam community is here to help.
```

Comment:
The tone here might be improved. For example, you might say something like:

If you have any questions about the these materials, please ask on the user's mailing list. Any and all questions are welcome.

We welcome any and all feedback on these materials to improve them over time. If you deliver a presentation based on these, we'd love to hear from you on the user's mailing list. Please share with us your thoughts and, if available and appropriate, a link to the recording. In particular, we'd like to know questions you've been asked to incorporate them back into the materials.